pywr/parameters/_parameters.pyx

```diff
@@ -670,6 +670,92 @@ cdef class ScenarioMonthlyProfileParameter(Parameter):
 
 ScenarioMonthlyProfileParameter.register()
 
+cdef class ScenarioWeeklyProfileParameter(Parameter):
+    """
+    This parameter provides a repeating annual profile with a weekly resolution. A
```

Comment:
Could you separate the docstring formatting into a single initial sentence on the first line, and then a following paragraph.